client/signup/steps/design-type-with-store/pressable-store/index.jsx

```diff
@@ -75,48 +74,80 @@ class PressableStoreStep extends Component {
+							{ translate( 'Start by entering your email address:' ) }
+						</FormLabel>
 						<div className="pressable-store__form-fields">
-							<FormTextInput ref={ (c) => this.onEmailInputRef(c) } isError={ this.state.error } isValid={ this.state.isValid } onChange={ this.onEmailChange } className="pressable-store__form-email is-spaced" type="email" placeholder="Email Address" name="email" />
-							<FormButton className="pressable-store__form-submit">{ this.props.translate( 'Get started on Pressable' ) } <Gridicon icon="external" size={ 12 } /></FormButton>
+							<FormTextInput
+								ref={ ( c ) => this.onEmailInputRef( c ) }
+								isError={ this.state.error }
+								isValid={ this.state.isValid }
+								onChange={ this.onEmailChange }
+								className="pressable-store__form-email is-spaced"
+								type="email"
+								placeholder="Email Address"
+								name="email"
+							/>
+							<FormButton className="pressable-store__form-submit">
+								{ translate( 'Get started on Pressable' ) }
+								<Gridicon icon="external" size={ 12 } />
+							</FormButton>
 						</div>
 						{ this.state.error && <FormInputValidation isError={ true } text={ this.state.error } /> }
 					</LoggedOutFormFooter>
 				</LoggedOutForm>
 				<LoggedOutFormLinks>
-					<LoggedOutFormLinkItem className="pressable-store__privacy-policy" target="__blank" href="https://pressable.com/legal/privacy-policy/">
-						{ this.props.translate( 'Pressable Privacy Policy', { comment: '“Pressable” is the name of a WordPress.org hosting provider' } ) } <Gridicon icon="external" size={ 12 } />
+					<LoggedOutFormLinkItem
+						className="pressable-store__privacy-policy"
+						target="__blank"
+						href="https://pressable.com/legal/privacy-policy/"
+					>
+						{ translate(
+							'Pressable Privacy Policy',
+							{ comment: '“Pressable” is the name of a WordPress.org hosting provider' }
+						) }
+						<Gridicon icon="external" size={ 12 } />
 					</LoggedOutFormLinkItem>
 				</LoggedOutFormLinks>
 			</div>
 		);
 	}
 
 	render() {
+		const { translate } = this.props;
+
 		return (
 			<div className="pressable-store">
 				<StepHeader
-					headerText={ this.props.translate( 'Create your WordPress Store' ) }
-					subHeaderText={ this.props.translate( 'Our partners at Pressable and WooCommerce are here for you.' ) }
+					headerText={ translate( 'Create your WordPress Store' ) }
+					subHeaderText={ translate( 'Our partners at Pressable and WooCommerce are here for you.' ) }
 				/>
 				{ this.renderStoreForm() }
 				<div className="pressable-store__back-button-wrapper">
-					<Button compact borderless onClick={ this.props.onBackClick }>
+					<Button compact={ true } borderless={ true } submitting={ false } onClick={ this.props.onBackClick }>
 						<Gridicon icon="arrow-left" size={ 18 } />
-						{ this.props.translate( 'Back' ) }
+						{ translate( 'Back' ) }
```

Comment:
I think this is generally used enough, but `Back` could be the back side of something or it could be a direction. we probably ought to have a `{ context: '…' }` here but I'm not sure which one we might already have to reuse